main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -99,25 +99,35 @@ DataTableCellUI.prototype._render = function() {
     } else {
       var arr = cell.v.split(" ");
       var spanArr = [];
-      for (var i=0; i<arr.length; i++){
+      for (var i=0; i<arr.length; i++) {
         if (URL.looksLikeUrl(arr[i])) {
           if (spanArr.length != 0) {
             var span = document.createElement('span');
             span.textContent = spanArr.join(" ");
             divContent.appendChild(span).appendChild(document.createTextNode('\u00A0'));
             spanArr = [];
           }
+          var [realURL, extra] = ((str) => {
+            var parts = str.split('\n');
+            return parts.length > 1 ? [parts[0], parts.slice(1).map((s) => ("\n" + s))] : [parts[0], []];
+          })(arr[i]);
```

Comment:
The IIFE used here does not really require any parameters as you can just use the arr[i] variable directly in it. 
I would imagine that it should look more like this

```js
var [realURL, extra] = (() => {
     var parts = arr[i]split('\n');
     return parts.length > 1 ? [parts[0], parts.slice(1).map((s) => ("\n" + s))] : [parts[0], []];
})();
```

Apart from that, looks everything looks fine to me. 👍🏾

Reply:
what about str, should I remove str also? @elroykanye

Reply:
Yes

Reply:
Done